code/src/App.js

```diff
@@ -1,9 +1,52 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+
+import { ThoughtsList } from './ThoughtsList';
+import { Form } from './Form';
+import { THOUGHTS_URL } from './urls';
 
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+	const [thoughts, setThoughts] = useState([]);
+
+	useEffect(() => {
+		fetchMessages();
+	}, []);
+
+	const fetchMessages = () => {
+		fetch(THOUGHTS_URL)
+			.then((res) => {
+				return res.json();
+			})
+			.then((data) => {
+				data.sort((a, b) => a.createdAt > b.createdAt);
+				const filteredMessages = data.filter((item) => item.message);
```

Comment:
I think you wanted to use `.map()`, since current `.filter()` method will return all messages (so, basically there is no difference between `data` and `filteredMessages`)